kinematics_interface_kdl/test/test_kinematics_interface_kdl.cpp

```diff
@@ -68,24 +69,25 @@ TEST_F(TestKDLPlugin, KDL_plugin_function) {
     ASSERT_TRUE(ik_->initialize(node_, end_effector_));
 
     // calculate end effector transform
-    std::vector<double> pos = {0, 0};
-    std::vector<double> end_effector_transform(16);
+    Eigen::Matrix<double, Eigen::Dynamic, 1> pos = Eigen::Matrix<double,2,1>::Zero();
```

Comment:
```suggestion
    Eigen::VectorXd pos(2);
```

Reply:
Can do the same in several other places  :+1:

Reply:
I believe that  `Eigen::VectorXd pos(2);` leaves `pos` uninitialized. I want the values to be zero in this case.

Reply:
Good idea to be explicit. Well, I think you can do this for short vectors. https://eigen.tuxfamily.org/dox/group__TutorialAdvancedInitialization.html

```suggestion
    Eigen::VectorXd pos(2) << 0, 0;
```

Or this for longer vectors:

https://books.google.com/books?id=g3JQDwAAQBAJ&pg=PA169&lpg=PA169&dq=Is+an+Eigen::Vector+initialized+to+zero&source=bl&ots=8-BXEhhGs3&sig=ACfU3U26Br8CgqJB1V2PYLcNr-wE5JTyuw&hl=en&sa=X&ved=2ahUKEwiG_vmLwLf5AhXnmWoFHVCoAKwQ6AF6BAgxEAM#v=snippet&q=initialize&f=false

Eigen::VectorXd pos(2) = Eigen::MatrixXd::Zeros(2,1);